tests/databases/redis/test_cluster.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import time
+
+import redis as redisdb
+
+from testsuite.databases.redis import service
+
+
+CLUSTER_INIT_TIMEOUT = 5.0
+
+
+@pytest.fixture(scope='session')
+def _redis_service_settings(pytestconfig):
+    return service.get_cluster_service_settings()
+
+
+def test_cluster_basic(redis_service, _redis_service_settings: service.ServiceSettings):
+    def cluster_nodes():
+        redis_db = redisdb.StrictRedis(
+            host=_redis_service_settings.host,
+            port=_redis_service_settings.master_ports[0],
+        )
+        return redis_db.cluster('NODES')
+
+    begin = time.time()
+    while time.time() - begin < CLUSTER_INIT_TIMEOUT:
```

Comment:
you dont need to wait. cluster must be initialized by startup script.

Reply:
tried to avoid the sleep hack, like in sentinel script, but i guess it can't be helped...